requirements-osbs.txt

```diff
@@ -87,7 +87,7 @@ mixpanel==4.5.0
 mock==3.0.5
 mockldap @ git+https://github.com/quay/mockldap.git@4265554a3d89fe39bf05b18e91607bec3fcf215a
 monotonic==1.5
-moto==1.3.14
+moto==2.0.1
```

Comment:
Remove this as it tries to bring in other deps and we don't need it at runtime. I've created PROJQUAY-1720 to track revisiting all the packages.